### ISSUE TITLE
Heading off confusion: When do headings fail WCAG? | TPG – The Accessibility Experts

### ISSUE THREAD

**Issue by studiomohawk** [frontendweekly/feeds#4557]
[Heading off confusion: When do headings fail WCAG? | TPG – The Accessibility Experts](https://developer.paciellogroup.com/blog/2020/03/heading-off-confusion-when-do-headings-fail-wcag/) /
